### PR TITLE
Add version to health check

### PR DESCRIPTION
**Pull request by iilei**
In order to make the api client more resilient I would like to read the release tag from someplace.

Thought the `/health` endpoint reasonable.

with this pr it responds with

```json
{
  "status": "ok",
  "version": "0.12.1"
}
```

Caution, I added [jq](https://stedolan.github.io/jq/) to the dependencies of the release script.

In case it is unfamiliar, please give it a chance.

From their readme:

> jq is like sed for JSON data - you can use it to slice and filter and map and transform structured data with the same ease that sed, awk, grep and friends let you play with text.

Screenshot of swagger header

### CODE REVIEW

**Comment by anthonynsimon** [comment]
Hey looks good! For the failing test you could just change the `expect({ status: 'ok' })` to something like:

```
.expect(res => {
    expect(res.body).toHaveExactProperties(['status', 'version']);
    expect(res.body.status).toEqual('ok');
});
```

**Comment by anthonynsimon** [comment]
I'm also not entirely sure if the `package.json` file where you're reading the version in the API be available on the dist build. Might need to add it to the build outputs.

**Comment by iilei** [comment]
> I'm also not entirely sure if the `package.json` file where you're reading the version from in the API be available on the dist build. Might need to add it to the build outputs.

I had the same thought so I already did a few changes (force-pushed)

Now there is another script available to be called with the version number you want to release:

`bin/version.sh 1.2.3`

which is a convenience helper and also utilized by the `release.sh` script.

I'll be revisiting this until weekend. Thanks for your swift response 🙂

**Comment by iilei** [comment]
I revisited the release script, now I think it's user friendly. Would you agree @anthonynsimon ?

File: bin/release.sh [hunk 1]
```diff
@@ -3,22 +3,34 @@
...
+    fi
+    return 0
+}
+
+function displayVersioningHint {
+    echo "Versions altered, you may fix this by running the following:"
+    echo ""
+    echo "  git add -u && git commit --amend --no-edit -a && bin/release.sh $RELEASE "
+    echo ""
+    echo "When preparing a new release, you can adit all package.json files at once via"
```
Comment (anthonynsimon) [inline bin/release.sh]: Just a minor typo on "...*edit* all package.json..." :)

Comment (iilei) [reply bin/release.sh]: didn't you know, _adit_ is slang for _audit_ and _edit_ 😄 
jk -fixed it

**Comment by anthonynsimon** [comment]
Hahah nice one :)

**Pull request merged**